canonical/ubuntu-pro-client
# fips: auto-upgrade to fips packages instead of hardcoding lists (SC-1745)

## Why is this needed?

See commit messages

Fixes: #2667
LP: #2060769




## Test Steps

On a {xenial,bionic,focal,jammy} {container,vm} try running
```
sudo env UA_FEATURES_FIPS_AUTO_UPGRADE_ALL=true pro enable fips-updates
```
Ensure that it upgrades the appropriate packages to their fips versions.


## Checklist


## Does this PR require extra reviews?
